index.html

```diff
@@ -4,7 +4,14 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
     <title>UI Library</title>
   </head>
-  <body></body>
```

Comment:
You have written body tag two times.

Reply:
Thankyou for reviewing. I will remove that